game.go

```diff
@@ -25,8 +25,42 @@ type GameListResult struct {
 type GameListIterator struct {
 	Iterator
 	Results chan Game
+	reverse bool
 }
 
+type Direction int
+
+const (
+    Forward Direction = iota
+    Backward
+)
+
+func (d Direction) Next(pager Pager) int {
+		var next int
+        switch d {
+        case Forward:
+                next = pager.Limit + pager.From
```

Comment:
backward case would still need a ```var next int``` so I think it's clearer to just do it at the top.